assets/xml/objects/object_in.xml

```diff
@@ -4,114 +4,124 @@
-        <Limb Name="object_in_Limb_013B0C" LimbType="Standard" Offset="0x13B0C"/>
-        <Limb Name="object_in_Limb_013B18" LimbType="Standard" Offset="0x13B18"/>
-        <Limb Name="object_in_Limb_013B24" LimbType="Standard" Offset="0x13B24"/>
-        <Limb Name="object_in_Limb_013B30" LimbType="Standard" Offset="0x13B30"/>
-        <Skeleton Name="object_in_Skel_013B88" Type="Flex" LimbType="Standard" Offset="0x13B88"/>
+        <Texture Name="gIngoShirtTLUT" OutName="shirt_tlut" Format="rgba16" Width="16" Height="16" Offset="0x1CD0"/>
+        <Texture Name="gIngoShoulderTLUT" OutName="shoulder_tlut" Format="rgba16" Width="16" Height="16" Offset="0x1ED0"/>
+        <Texture Name="gIngoUnusedEyeTex" OutName="unused_eye" Format="rgba16" Width="32" Height="32" Offset="0x2150"/>
+        <Texture Name="gIngoShoulderGradient" OutName="shoulder_gradient" Format="ci8" Width="8" Height="8" Offset="0x2950" TlutOffset="0x1ED0"/>
+        <Texture Name="gIngoArmHairTex" OutName="arm_hair" Format="ci8" Width="16" Height="32" Offset="0x2990" TlutOffset="0x1CD0"/>
+        <Texture Name="gIngoFingersTex" OutName="fingers" Format="ci8" Width="16" Height="16" Offset="0x2B90" TlutOffset="0x1CD0"/>
+        <Texture Name="gIngoShirtSleeveTex" OutName="sleeve" Format="ci8" Width="8" Height="8" Offset="0x2C90" TlutOffset="0x1CD0"/>
+        <Texture Name="gIngoShirtButtonsTex" OutName="shirt_buttons" Format="ci8" Width="32" Height="32" Offset="0x2CD0" TlutOffset="0x1CD0"/>
+        <Texture Name="gIngoShirtTex" OutName="shirt" Format="ci8" Width="8" Height="8" Offset="0x30D0" TlutOffset="0x1CD0"/>
+        <Texture Name="gIngoShirtStainTex" OutName="shirt_stain" Format="ci8" Width="8" Height="8" Offset="0x3110" TlutOffset="0x1CD0"/>
+        <Texture Name="gIngoPocketTex" OutName="pocket" Format="ci8" Width="8" Height="8" Offset="0x31D0" TlutOffset="0x1CD0"/>
+        <Texture Name="gIngoHeadTLUT" OutName="head_tlut" Format="rgba16" Width="252" Height="1" Offset="0x3210"/>
+        <Texture Name="gIngoTLUT2" OutName="tlut2" Format="rgba16" Width="100" Height="1" Offset="0x3408"/>
+        <Texture Name="gIngoHeadGradient2" OutName="head_gradient_2" Format="ci8" Width="8" Height="8" Offset="0x34D0" TlutOffset="0x3210"/>
+        <Texture Name="gIngoEarTex" OutName="ear" Format="ci8" Width="8" Height="8" Offset="0x3510" TlutOffset="0x3408"/>
+        <Texture Name="gIngoBrownTex" OutName="brown" Format="ci8" Width="8" Height="8" Offset="0x3550" TlutOffset="0x3408"/> <!-- Used in his mustache, eyebrows, pitchfork -->
+        <Texture Name="gIngoEyebrowEdgeTex" OutName="eyebrow_edge" Format="rgba16" Width="16" Height="16" Offset="0x3990"/>
+        <Texture Name="gIngoHairTex" OutName="hair" Format="ci8" Width="16" Height="16" Offset="0x3B90" TlutOffset="0x3408"/> <!-- Top of head -->
+        <Texture Name="gIngoHair2Tex" OutName="hair_2" Format="ci8" Width="16" Height="16" Offset="0x3C90" TlutOffset="0x3408"/> <!-- Side of head -->
```

Comment:
```suggestion
        <Texture Name="gIngoHairTex" OutName="hair" Format="ci8" Width="16" Height="16" Offset="0x3B90" TlutOffset="0x3408"/>
        <Texture Name="gIngoSideburnsTex" OutName="sideburns" Format="ci8" Width="16" Height="16" Offset="0x3C90" TlutOffset="0x3408"/>
```

<details>

<summary>
Screenshot of tris that use the sideburns image
</summary>

![](https://421.es/doyu/27e4w1)

</details>